ui/src/main/scripts/plugins/apm/ts/apmPlugin.ts

```diff
@@ -30,14 +30,75 @@ module APM {
       .id(APM.pluginName)
       .title(() => 'Application Performance')
       .href(() => '/hawkular-ui/apm')
+      .subPath('Components', 'components', builder.join(APM.templatePath, 'apm.html'))
+      .subPath('Distributed Tracing', 'tracing', builder.join(E2E.templatePath, 'e2e.html'))
+      .subPath('Business Transactions', 'btm', builder.join(BTM.templatePath, 'btm.html'))
       .rank(30)
       .build();
     builder.configureRouting($routeProvider, tab);
     $locationProvider.html5Mode(true);
     $routeProvider.
-      when('/hawkular-ui/apm', {
+      when('/hawkular-ui/apm/components', {
         templateUrl: 'plugins/apm/html/apm.html',
         controller: 'APM.APMController'
+      }).
+      when('/hawkular-ui/apm/tracing', {
+        templateUrl: 'plugins/e2e/html/e2e.html',
+        controller: 'E2E.E2EController'
+      }).
+      when('/hawkular-ui/apm/btm', {
+        templateUrl: 'plugins/btm/html/btm.html',
+        controller: 'BTM.BTMController'
+      }).
+      when('/hawkular-ui/apm/btm', {
```

Comment:
Ah sorry, misunderstood. Thought you were referring to the fact that the subpath (line 33-35) were being duplicated by the route/when code.

I'll remove the duplicate route.